extension/src/candlestick.rs

```diff
@@ -11,189 +15,175 @@ use crate::{
 };
 use tspoint::TSPoint;
 
-#[pg_schema]
-pub mod toolkit_experimental {
-    use super::*;
-
-    flat_serialize_macro::flat_serialize! {
-        #[derive(Serialize, Deserialize, Debug, Copy)]
-        enum VolKind {
-            unused_but_required_by_flat_serialize: u64,
-            Missing: 1 {},
-            Transaction: 2 { vol: f64, vwap: f64 },
-        }
+flat_serialize_macro::flat_serialize! {
```

Comment:
Uff, github did not do a great job on this portion of the diff.  Looks like there wasn't anything changed beyond removing the `toolkit_experimental` and `cargo fmt` changes, but let me know if I missed something I should have looked closer at.